html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <a href="/html-relative-links/homePage.html"><li>Home Page</li></a>
```

Comment:
so while this does work, the format you want to go with here is: 
```javascript
href="../index.html"
```

The goal is to route from the current page to the page you want to route to, not to route from the parent directory